js/app.js

```diff
@@ -0,0 +1,43 @@
+$(function(){
+
+	var temperature = 0
+
+		$("#entry").submit(function(){
+
+		event.preventDefault();
+		//console.log(event);
+		//console.log('form was submitted');
+
+		//read amount entered in (this will be a string)
+		var enteredAmount = $("#celsius").val();
+
+		// convert string amount to integer
+		var parsedAmount = parseInt(enteredAmount);
+
+		//add the amount to the total
+		temperature = parsedAmount * 9 / 5 + 32;
+
+		console.log('celsius', temperature)
+
+		//print updated total to screen and keeps two decimal places and the $ sign
+		$("#temp").text(temperature + " F");
+		//clear text field
+		$("#celsius").val("");
+	
+
+		if(temperature > 90){
+			$('body').css('background', 'url(images/fire.jpg)');
```

Comment:
@mbhat25 If you add `background-size: cover` in your css declarations, you are able to view the  whole image. Or you could have added that rule in a class and added that class if it meet the condition.